docs/pipelines.md

```diff
@@ -201,6 +202,26 @@ The resource `my-image` is expected to be given to the `deploy-app` `Task` from
 the `build-app` `Task`. This means that the `PipelineResource` `my-image` must
 also be declared as an output of `build-app`.
 
+## Labels
```

Comment:
@bobcatfish At this point all of `Task`, `TaskRun`, `Pipeline`, and `PipelineRun` have a `Labels` section with a lot of duplicated info. Maybe better to create a new file to avoid the repetition? Thoughts?

Reply:
Done, moved to `labels.md`.